xml/art_admin_slemicro_cockpit.xml

```diff
@@ -125,945 +140,2095 @@
+    If you created a block logical volume, you need to format the volume by
+    clicking <guimenu>Format</guimenu> and filling the details as described in
+    <xref linkend="list-format-partition"/>.
+   </para>
+   <para>
+    If you created a pool of thinly provisioned volumes, you can create a thin
+    volume by clicking <guimenu>Create thin volume</guimenu>.
+   </para>
+   <figure xml:id="fig-thinly-volumes">
+    <title>Creating a thinly provisioned volume</title>
+    <mediaobject>
+     <imageobject role="fo">
+      <imagedata  fileref="cockpit_thin_volumes.png" width="100%"/>
+     </imageobject>
+     <imageobject role="html">
+      <imagedata fileref="cockpit_thin_volumes.png" width="100%"/>
+     </imageobject>
+     <textobject role="description"><phrase>Volume thin</phrase>
+     </textobject>
+    </mediaobject>
+   </figure>
+   <para>
+    Enter a unique name, select the size of the volume, and confirm by clicking
+    <guimenu>Create</guimenu>. You can create several volumes of the particular
+    volume group by clicking <guimenu>Create thin volume</guimenu> again.
+    Format the volumes by clicking <guimenu>Format</guimenu> and filling the
+    details as described in <xref linkend="list-format-partition"/>.
+   </para>
+   <para>
+    Other actions on a particular volume are available in the threedot menu:
+    <guimenu>Deactivate</guimenu>, <guimenu>Create snapshot</guimenu> or
+    <guimenu>Delete</guimenu>.
+   </para>
+   <para>
+    Shrinking or growing a volume is available by clicking on
+    <guimenu>Shrink</guimenu> or <guimenu>Grow</guimenu> in the particular
+    volume details.
+   </para>
+  </section>
+ </section>
+ <section xml:id="sec-cockpit-networking">
+  <title>Managing networking</title>
+  <para>
+   After clicking <guimenu>Networking</guimenu>, you can view traffic on your
+   system, manage firewall, manage network interfaces or view network logs.
+  </para>
+  <figure xml:id="fig-networking">
+   <title>Networking</title>
+   <mediaobject>
+    <imageobject role="fo">
+     <imagedata  fileref="cockpit_networking_plugin.png" width="100%"/>
+    </imageobject>
+    <imageobject role="html">
+     <imagedata fileref="cockpit_networking_plugin.png" width="100%"/>
+    </imageobject>
+    <textobject role="description"><phrase>Cockpit networking</phrase>
+    </textobject>
+   </mediaobject>
+  </figure>
+  <section xml:id="sec-networking-firewall">
+   <title>Firewall</title>
+   <important>
+    <title>Firewall and Podman</title>
+    <para>
+     Using firewall along with Podman may result in missing Podman-relateds
+     firewall rules after reloading the firewalld service. Therefore, it is
+     recommended to keep the firewall in its default setting (disabled) if you
+     intend to use Podman.
+    </para>
+   </important>
+   <para>
+    The <guimenu>Firewall</guimenu> view enables you turn on the firewall,
+    which is off by default. After turning it on, you can edit firewall rules
```

Comment:
Maybe worth adding a note about possible loosing of connectivity when enabling firewall as described here: https://bugzilla.suse.com/show_bug.cgi?id=1202278 and https://github.com/cockpit-project/cockpit/issues/17660